README.rst

```diff
@@ -9,6 +9,12 @@ p4c-bm
 Generates the JSON configuration for the `behavioral-model (bmv2)
 <https://github.com/p4lang/behavioral-model>`_.
 
+NOTE: This repository is no longer under active development or
+maintenance.  We recommend that you instead consider using the
+currently developed open source P4 tools, which are
+[`p4c`](https://github.com/p4lang/p4c) and
```

Comment:
The links are not rendered properly because for some historical reasons, I chose to make this README a rst document and not a markdown one.

Reply:
I have updated the markup and previewed it in my fork's branch, and it looks better now.